project_issue_task/models/project_issue.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 - 2013 Daniel Reis
+# Copyright 2016 - Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, models, _
+
+
+class ProjectIssue(models.Model):
+    _inherit = 'project.issue'
+
+    @api.multi
+    def action_create_task(self):
+        """
+        Create and a related Task for the visit report, and open it's Form.
+        """
+        res = {}
+        for rec in self:
+
+            assert not rec.task_id, _("A Task is already assigned to the "
+                                      "Issue!")
+
+            rec_fields = ['project_id', 'analytic_account_id', 'location_id']
+            task_data = dict([(x, getattr(rec, x).id) for x in rec_fields
```

Comment:
Maybe it's better to just assign directly data for such small number of fields. Please encapsulate this in a `_prepare_...` method.